src/core/binding.c

```diff
@@ -167,6 +167,10 @@ QuicBindingInitialize(
     *NewBinding = Binding;
     Status = QUIC_STATUS_SUCCESS;
 
+    CxPlatDispatchLockAcquire(&Binding->StatelessOperLock);
```

Comment:
A lock might be a bit too extreme here, but we do need a memory barrier here somehow, otherwise platforms like arm will have an issue. We don't currently have any CXPLAT code for explicit memory barriers.

Reply:
can't use interlocked exchange?

Reply:
Yeah I can do that and make it not a bitfield. I might just do that.